### PR TITLE
Cert command

### PR DESCRIPTION
**Pull request by miklosduma**
Added `sparkl cert USER PASSWORD [--output OUTPUT] [--format FORMAT]` command to download certificates in either pem or pkcs12 format

### CODE REVIEW

File: sparkl_cli/cmd_cert.py [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+        help="user password or access token secret")
+    subparser.add_argument(
+        "-o", "--output",
+        type=str,
+        help="write certificate to named local file")
+    subparser.add_argument(
+        "-f", "--format",
+        help="defaults to 'pem'. Use 'pem' for CLI, 'p12' for browsers",
+        type=str,
+        default=CERT_TYPES[0],
```
Comment (jacobythwaites) [inline sparkl_cli/cmd_cert.py]: Very nice

File: sparkl_cli/cmd_cert.py [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+        choices=CERT_TYPES)
+
+
+def cert_buffer(response):
+    """Generator object that returns the content of \
```
Comment (jacobythwaites) [inline sparkl_cli/cmd_cert.py]: I'd prefer the triple quote on its own line

File: sparkl_cli/cmd_login.py [hunk 3]
```diff
@@ -17,14 +17,13 @@
...
-import getpass
-
 from sparkl_cli.CliException import (
     CliException)
 
 from sparkl_cli.common import (
     del_current_folder,
-    sync_request)
+    sync_request,
+    get_password)
```
Comment (jacobythwaites) [inline sparkl_cli/cmd_login.py]: Cos used by cert, I see.

File: sparkl_cli/test/test_login_logout.py [hunk 4]
```diff
@@ -27,19 +27,22 @@
 
 class Tests():
 
-    def test_login(self):
-        sparkl(
-            "connect",
-            url=TEST_SSE)
+    def test_login(self, connect_sparkl_setup):
```
[the remark this answers is not in the document]
Comment (miklosduma) [reply sparkl_cli/test/test_login_logout.py]: Yes. In pytest you can create test fixtures in a dedicated file called conftest.py. On the long run I would like to move most of the commonly used test setup/teardown methods there. It makes no sense to have the same logic in 20x test files :)

File: sparkl_cli/cmd_cert.py (hunk 2 above)
Comment (miklosduma) [reply sparkl_cli/cmd_cert.py]: Woops, sorry, in a previous project I was using [Sphinx](https://sphinx-rtd-tutorial.readthedocs.io/en/latest/docstrings.html) for auto-generated docs and got way too used to it :)

**Pull request merged**